controller_manager:
  ros__parameters:  # <-- Two underscores now
    update_rate: 30
    use_sim_time: true
    diff_cont:
      type: diff_drive_controller/DiffDriveController
    joint_broad:
      type: joint_state_broadcaster/JointStateBroadcaster

diff_cont:
  ros__parameters:  # <-- This was already correct
    publish_rate: 30.0
    base_frame_id: base_link
    left_wheel_names: ['left_wheel_joint']
    right_wheel_names: ['right_wheel_joint']
    wheel_separation: 0.15
    wheel_radius: 0.068
    use_stamped_vel: false
